code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -755,3 +755,16 @@
 	icon_state = "lean"
 	list_reagents = list(/datum/reagent/consumable/lean = 20)
 	random_sprite = FALSE
+
+/obj/item/reagent_containers/food/drinks/colocup/lean/suicide_act(mob/living/carbon/user)
+	if(!reagents.total_volume)
+		user.visible_message(span_warning("[user] tries to chug the cup of lean... There is no lean left!"))
+		return SHAME
+	else
+		user.say("I LOVE LEAN!!")
```

Comment:
you should add forced = "lean suicide" or something similar here to bring it in line with other forced instances of say()
```suggestion
		user.say("I LOVE LEAN!!", forced = "lean suicide")
```

Reply:
yes, it will help avoid incorrect automutes and admin bans.